biztalk/adapters-and-accelerators/lob-and-enterprise-2020-support.md

```diff
@@ -0,0 +1,145 @@
+
+> [!NOTE]
+> There is no seperate installation for BizTalk Adaper Pack (BAP). These are installed with BizTalk Server 2020 release onwards. 
+
+## Supported LOB Systems
+
+### SAP Server
+
+| SAP Server | Supported |
+| --- | --- |
+| SAP ECC 6.0 Unicode | Yes |
+| SAP ECC 6.0 Unicode with EHP 4.0 | Yes |
+| SAP ECC 6.0 with EHP 7.0 and all EHP previous versions | Yes |
+| SAP ECC 6.0 with EHP 7.5 | Yes |
+| SAP ECC 6.0 with EHP 8.0 | Yes |
+
+### SAP Client
+
+| SAP Client | Supported |
+| --- | --- |
+| SAP .NET Connector (NCo) 3.0 | Yes |
+
+### Oracle Database
+
+| Oracle Database | Supported |
+| --- | --- |
+| Oracle database 12.1 | Yes |
+| Oracle database 18c (18.3) | Yes |
+| Oracle database 19c (19.3) | Yes |
+
+### Oracle Client
+
+| Oracle Client | Supported |
+| --- | --- |
+| Oracle Database 12.1 Client | Yes |
+| Oracle Database 18c Client (18.3) | Yes |
+| Oracle Database 19c Client (19.3) | Yes |
+
+### Oracle E-Business Suite
+
+| Oracle E-Business Suite | Supported |
+| --- | --- |
+| Oracle EBS 12.1 | Yes |
+| Oracle EBS 12.1.3 | Yes |
+| Oracle EBS 12.2.5 | Yes |
+
+### Siebel Server
+
+| Sieble Server | Supported |
+| --- | --- |
+| Siebel 17 | Yes |
+
+### Sieble Client
+
+| Sieble Client Version | Supported |
+| --- | --- |
+| Siebel Web Client 17 | Yes |
+
+### SQL Server
+
+| SQL Server Version | Supported |
+| --- | --- |
+| SQL Server 2014 | Yes |
+| SQL Server 2016 | Yes |
+| SQL Server 2017 | Yes |
+| SQL Server 2019 | Yes |
+| Azure SQL Database | Yes |
+
+> [!IMPORTANT]
+> This refers to the SQL Server versions used by the WCF-SQL adapter in BizTalk Server 2020. For the SQL Server versions supported as the back-end system to BizTalk Server, refer to Hardware and Software Requirements for BizTalk Server 2020 [here](..\install-and-config-guides\hardware-and-software-requirements-for-biztalk-server-2020.md)
+
+## Supported Enterprise Systems
+
+### JD Edwards Enterprise One
+
+| JD Edwards EnterpriseOne Application Release | Supported |
+| --- | --- |
+| 9.2 | Yes |
+| 9.1 | Yes |
+
+
+| JD Edwards EnterpriseOne Tools Release | Supported |
+| --- | --- |
+| 9.2 | Yes |
+| 9.1 | Yes |
+
+### People Soft Enterprise
+
+| PeopleSoft Enterprise Application Release | Supported |
+| --- | --- |
+| 9.0 | Yes |
+
+
+| PeopleSoft Enterprise Tools Release | Supported |
+| --- | --- |
+| 8.49 | Yes |
+
+### TIBCO Rendezvous
+
+| TIBCO Rendezvous | Supported |
+| --- | --- |
+| 8.4.6 | Yes |
+
+
+| TIBCO Rendezvous Client | Supported |
+| --- | --- |
+| 8.4.3 | Yes |
+
+### TIBCO Enterprise Message Service
+
+| TIBCO EMS | Supported |
+| --- | --- |
+| 8.5 | Yes |
+
+> [!IMPORTANT]
+> BizTalk 2020 Onwards TIBCO Enterprise Message Service Adapter is only supported in 64 bit mode. Please configure Tibco Enterprise Message Service Adapter to use 64 bit host.
```

Comment:
Applicable starting BizTalk Server 2020, TIBCO Enterprise Message Service Adapter is supported in 64 bit host only.